src/api/query.rs

```diff
@@ -251,117 +251,112 @@ impl Client {
     }
 
     /// Returns bucket measurements
-    pub async fn list_measurements(&self, bucket: &str) -> Result<Vec<String>, RequestError> {
-        let req_url = self.url("/api/v2/query");
+    pub async fn list_measurements(
+        &self,
+        bucket: &str,
+        days_ago: Option<i64>,
```

Comment:
I'm a bit late to this PR, but I'd like to note that this introduces a backwards incompatible change to anyone using one of these 3 functions.

Backwards compatibility could be maintained by restoring the original functions and adding new functions with the extra param. Alternatively (and this would be my preference) we could mention it in the release notes

Reply:
@Boudewijn26 that's a valid point, I'll yank version 0.4.6 and bump minor version to v0.5 and create a release note, bumping major version seems to be unnecessary since we haven't reach v1 yet.

Also a clean up seems necessary like perhaps introducing both `start` and `stop` optional parameter so that it more closely resembles the API documentation.

I'll do this either today or tomorrow.